src/alert.cpp

```diff
@@ -1900,7 +1895,7 @@ namespace libtorrent {
 		, m_alloc(alloc)
 		, m_num_peers(int(peers.size()))
 	{
-		std::size_t total_size = m_num_peers; // num bytes for sizes
+		std::size_t total_size = 0; // num bytes for sizes
 		for (int i = 0; i < m_num_peers; i++) {
 			total_size += peers[i].size();
```

Comment:
looking at this, is ``size()`` really correct here? I worry that it means ``sizeof(in_addr)`` or ``sizeof(in6_addr)``, rather than 6 or 18.